documentation/usersguide/livestatus.md

```diff
@@ -10,6 +10,202 @@ Livestatus is one of the standard APIs for Naemon. It can be used to query live
 status and configuration information from the Naemon core.
 Detail description is available on <a href="http://mathias-kettner.de/checkmk_livestatus.html">mathias-kettner.de</a>.
 
+### Module Options
+
+The livestatus module is usually loaded from the `/etc/naemon/module-conf.d/livestatus.cfg` file:
+
+    broker_module=/usr/lib64/naemon/naemon-livestatus/livestatus.so debug=0 /var/cache/naemon/live
+    event_broker_options=-1
+
+The module accepts a number of options:
+
+#### data_encoding
+
+Changes encoding style.
+
+Possible values:
+
+- utf8 *(Default)*
+- latin1
+- mixed
+
+ex.:
+
+    broker_module=.../livestatus.so data_encoding=utf8 ...
+
+#### debug
+
+Enable/disable debug logging.
+
+Possible values:
+
+- 0 debug logging off *(Default)*
+- 1 debug logging on
+- 1
```

Comment:
extra `1` (when I made the suggestion I couldn't figure out how to make it span multiple lines..)